src/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,194 @@
+import React, { FC, useState, forwardRef } from 'react';
+import { keyframes, styled } from '@storybook/theming';
+import * as Acc from '@radix-ui/react-accordion';
+import { color, fontFamily, fontSize, spacing } from '../_tokens';
+import { Icon } from '../Icon';
+import type { Icons } from '../Icon/Icon';
+
+export interface AccordionProps {
+  children: React.ReactNode | string;
+  defaultValue?: string | undefined;
+  iconName?: Icons | false;
+  iconSize?: number;
+  id?: string | undefined;
+  inverse?: boolean;
+  value?: string;
+  triggerCopy?: string;
+}
+
+const slideDown = keyframes`
+  from {
+    height: 0;
+  }
+  to {
+    // dynamically set height
+    height: var(--radix-accordion-content-height);
+  }
+}`;
+
+const slideUp = keyframes`
+  from {
+    height: var(--radix-accordion-content-height);
+  }
+  to {
+    height: 0;
+  }
+}`;
+
+// Outermost accordion container
+const AccordionWrapper = styled(Acc.Root)<{
+  inverse?: boolean;
+}>`
+  border: 1px solid;
+  border-bottom: 0;
+  border-color: ${({ inverse }) =>
+    inverse ? color.whiteTr10 : color.slate200};
+  border-radius: 5px;
+  color: ${({ inverse }) => (inverse ? color.white : color.slate800)};
+  font-family: ${fontFamily.sans};
+  font-size: ${fontSize[16]};
+`;
+
+// Wrapper for each accordion item's trigger & content
+const Item = styled(Acc.Item)<{
+  inverse?: boolean;
+}>`
+  border-bottom: 1px solid;
+  border-color: ${({ inverse }) =>
+    inverse ? color.whiteTr10 : color.slate200};
+  flex-flow: column nowrap;
+  overflow: hidden;
+  padding: ${spacing[2]} ${spacing[8]};
+
+  &:last-of-type {
+    border-bottom-left-radius: 4px;
+    border-bottom-right-radius: 4px;
+  }
+
+  &:focus-within {
+    position: relative;
+    z-index: 1;
+    box-shadow: 0 0 0 2px var(--mauve-12);
+  }
+`;
+
+// the button controlling the open/closed state of each item
+const Trigger = styled(Acc.Trigger)<{
+  inverse?: boolean;
+}>`
+  align-items: center;
+  background: none;
+  border: 0;
+  color: ${({ inverse }) => (inverse ? color.white : color.slate800)};
+  cursor: pointer;
+  display: flex;
+  font-size: ${fontSize[16]};
+  font-weight: 400;
+  justify-content: space-between;
+  padding: ${spacing[2]} 0;
+  transition: all 0.16s ease-in-out;
+  width: 100%;
+
+  &:hover,
+  &:focus {
+    color: ${({ inverse }) => (inverse ? color.slate50 : color.slate700)};
+    transform: translateY(-1px);
```

Comment:
I would remove this transition. We only use that for links. Perhaps you could change the background color on hover and focus.